test/data/expected/ComplexRecord.ts.test

```diff
@@ -8,7 +8,7 @@ export interface EmailAddress {
     dateAdded: number;
 }
 
-export interface User {
+export interface comexampleavroUser {
```

Comment:
assuming that the namespace was `com.example.avro` shouldn't the name be `comExampleAvroUser`?

Reply:
The transformName function in the tests removes the dots.